### ISSUE TITLE
Bug Report: GitHub release builds are prototypes?

### ISSUE THREAD

**Issue by swordcube** [FunkinCrew/Funkin#3672]
### Issue Checklist

- [X] I have properly named my issue
- [X] I have checked the Issues/Discussions pages to see if my issue has already been reported

### Platform

Compiled from GitHub Source Code

### Browser

None

### Version

0.5.2

### Description (include any images, videos, errors, or crash logs)

If you download 0.5.2 from GitHub releases, in the main menu AND gameplay it will say this is a prototype build, which i assume is unintentional
![image](https://github.com/user-attachments/assets/55cae900-02ea-4e3a-bb50-848dac396119)

I haven't seen anyone else report this yet, and the build on itch.io is ACTUALLY 0.5.2:
![image](https://github.com/user-attachments/assets/865ee889-86e2-4fde-8e2f-b818d4874b4d)

I had to select "Compiled from GitHub source code" because there is literally no option for GitHub release builds

### Steps to Reproduce

1. Download 0.5.2 from GitHub releases
2. Go to main menu and look at the bottom left corner

**Comment by NotHyper-474** [comment]
I assume they used automated builds and forgot to remove the GITHUB_BUILD define so they ended up being uploaded as if they were debug builds.

**Comment by swordcube** [comment]
> I assume they used automated builds and forgot to remove the GITHUB_BUILD define so they ended up being uploaded as if they were debug builds.

Would make sense but isn't it also strange how the version number didn't change from 0.5.1?

**Comment by NotHyper-474** [comment]
> Would make sense but isn't it also strange how the version number didn't change from 0.5.1?

The commit hash points to e66290c55f7141402223644f06ec8a69edeee089 so technically it is 0.5.2, still odd how it's not the latest commit

**Comment by Cartridge-Man** [comment]
Pretty sure this just says prototype seeing as how the developers almost always call this game a prototype at the moment, hence why it’s a game that isn’t quite finished in their eyes and is still actively being worked on with new weeks, new features, etc

**Comment by swordcube** [comment]
> Pretty sure this just says prototype seeing as how the developers almost always call this game a prototype at the moment, hence why it’s a game that isn’t quite finished in their eyes and is still actively being worked on with new weeks, new features, etc

Well yes that makes sense but consider this:
No build starting from v0.3 til v0.5.2 has had this watermark

And this is *only* in the GitHub releases, the itch.io build says the proper version and isn't labeled as prototype

**Comment by Braite1** [comment]
I also noticed that in the github version of 0.5.2, Darnell BF Mix seems to automatically use botplay, idk if that's just me

**Comment by Hundrec** [comment]
> I also noticed that in the github version of 0.5.2, Darnell BF Mix seems to automatically use botplay, idk if that's just me

Just downloaded and opened the GitHub build to test this out and got my save file snapped like https://github.com/FunkinCrew/Funkin/issues/3679

**Comment by Braite1** [comment]
> Just downloaded and opened the GitHub build to test this out and got my save file snapped like #3679

Yeah that happened to me too, except it was for downgrading in my case

**Comment by Beelijah** [comment]
yeah, this happened to me while i was making a mod. swapped around tons of the assets, uploaded my build and realized my hours of work was on a prototype build the whole time ;-;
